### PR TITLE
#158239862 Create profile page

### PR DESCRIPTION
**Pull request by micahoriaso**
**What does this PR do?**
Create user profile page

**Description of the task to be completed**
Build the user interface elements for the user profile page

**How to manually test**
* Clone the repo and cd into it.
* checkout into ft-user-profile-page-158239862 branch
* cd into the UI folder
* open user-profile.html with a browser

**Relevant pivotal tracker stories**
#158239862

**Screenshots**
fireshot capture 007 - ride detail - file____home_oriaso_desktop_python_ride-my-way_ui_profile html

### CODE REVIEW

**Comment by micahoriaso** [comment]
Hi @wandesky what do you think of this update?

File: UI/profile.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+                                    <div class="row">
+                                        <div class="eight wide column">
+                                            <div class="form-field">
+                                                <label class="field-label" for="first_name">First Name</label>
+                                                <input type="text" value="Micah" name="first_name" readonly>
+                                            </div>
+                                        </div>
+                                        <div class="eight wide column">
+                                            <div class="form-field">
+                                                <label class="field-label" for="last_name">Last name</label>
```
Comment (wandesky) [inline UI/profile.html:59]: Change to Last Name so that it matches First Name

**Review by wandesky — APPROVED** [review]
@micahoriaso the page looks awesome. I hope you will attend the Open Skills session this coming Saturday so that you show me how to make such an epic page.

File: UI/profile.html (hunk 1 above)
Comment (micahoriaso) [reply UI/profile.html:59]: Good catch. It's fixed

fireshot capture 009 - ride detail - file____home_oriaso_desktop_python_ride-my-way_ui_profile html

Comment (micahoriaso) [reply UI/profile.html:59]: I will be attending the session. Looking forward to meeting you there.

**Pull request merged**